ext/ldap/ldap.c

```diff
@@ -248,7 +248,7 @@ static bool php_ldap_is_numerically_indexed_array(zend_array *arr)
 		}
 	} ZEND_HASH_FOREACH_END();
 
-	return false;
+	return true;
```

Comment:
ah that s unfortunate !

Reply:
I bet this happened during the downwards merge. This was fixed first in master, and then needed a slightly different patch on 8.3-8.4. It's right on master but wrong on 8.3-8.4, so something must've gone wrong while merging